src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const TextContainer = styled.div`
+  display: inline-block;
+  flex-direction: column;
+  font-weight: bold;
+`;
+
+export const TextComponent = styled.div`
+  margin-top: 10px;
```

Comment:
its best to use `rem` instead of `px`.[ this article summarizes](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px) this concept pretty well. implementation-wise, 1rem = 16px, so this^ would be `margin-top: .625rem`

Reply:
Since I am adjusting it to just `export const TextContainer = styled.p`, would I still need to specify the margins? 🤔

Reply:
the default margins that come with a `p` are probably rendering, yeah? id override those and match the margin-top to what is in the mocks

Reply:
Yep! Ignore me 🙈 Turns out I was looking at the wrong component! This makes sense.